### ISSUE TITLE
Reimagine the design of the Hub

### ISSUE THREAD

**Issue by meiqimichelle** [18F/hub#213]
If there is a designer with spare cycles, I would be happy to re-imagine the Hub design again now that we have learned more about where we're going and what people what to use it for. Ping me if you want background (sorry that I don't have time to write a nice issue for you now!) :(

**Comment by meiqimichelle** [comment]
Some issues:
- now that we've learned how people are using the site, we have outgrown the current site nav. Re-assess content structure.
- think about ways to facilitate browsing through the tons of info. 'How-to' may not be the way to go.

**Comment by mbland** [comment]
- [ ] Ensure every page has a `title:` in its front matter.
- [ ] Add `tags:` to front matter where appropriate.
- [ ] Create a tool/plugin for discovering missing titles (and tags?).
- [ ] Follow-through on @jenniferthibault's design (implicated in #269).
- [ ] Draft content-first design a la Stephanie Hay (h/t @brethauer)

**Comment by meiqimichelle** [comment]
Perhaps we should move the first three of those tasks to another issue. They don't relate to re-imagining the design of the Hub. What do you think @mbland ?

**Comment by mbland** [comment]
@meiqimichelle Makes sense, though #274 takes care of the first and third. Checked 'em off here, but will create a new issue.

**Comment by meiqimichelle** [comment]
:+1:
